src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -545,6 +551,16 @@ class ProjectReview extends Component {
     });
   };
 
+  requestClarification = () => {
+    this.setState({
+      showRequestClarification: !this.state.showRequestClarification
+    });
+  };
+
+  closeClarificationModal = () => {
+    this.setState({ showRequestClarification: !this.state.showRequestClarification });
+  };
```

Comment:
These state updaters are duplicated here and in `ConsentGroupReview.js`. It would be nice if we could abstract these into one place.